charts/bria/values.yaml

```
@@ -22,6 +22,8 @@ bria:
     service:
       type: ClusterIP
       port: 2742
+      staticIp: ""
```

```suggestion
      staticIP: ""
```
We use uppercase `P` in other charts with this config